packages/sit-onyx/src/components/OnyxIcon/types.ts

```diff
@@ -6,11 +6,14 @@ export type OnyxIconProps = {
    * not be sanitized.
    */
   icon: string;
-  /** Icon size. */
+  /**
+   * Icon size in px. Will be translated to the according rem value.
+   * @default 24
+   */
   size?: IconSize;
   /** Icon color. */
   color?: OnyxColor | "currentColor";
 };
 
-export const ICON_SIZES = ["2xs", "xs", "sm", "md", "lg", "xl", "2xl"] as const;
+export const ICON_SIZES = ["12", "16", "24", "32", "48", "64", "96"] as const;
```

Comment:
I am still in favor of adding the "px" unit. A number without unit is meaningless. We have pixels in figma and these are pixel values, why hide that fact?

Reply:
Fine for me :)

Reply:
ok ^^ thought the description in the docs is enough ^^ (also I like the ability on project side to globally search for "px" to find out where px is used instead of rem ¯\\_(ツ)_/¯

Reply:
you know what, I'll offer both: the ones with px and the ones with rem :)
because if (public) users have figma in dev mode set to rem, they'll see
![image](https://github.com/SchwarzIT/onyx/assets/151019977/0a8afe97-02b4-4925-9d3e-c9e78dca75f4)

Reply:
Hmm I would not offer both...then you would have 2 options that do the same thing. We should decide on one

Reply:
I am also fine with keeping the t-shirt sizes, I mean we also use them everywhere else like in the spacing tokens, border radius tokens etc

Reply:
just need to recreate the screenshots now :)

Reply:
![image](https://github.com/SchwarzIT/onyx/assets/151019977/e8bc5dc4-ff70-4b76-a0d8-aca4e00588e4)

with supporting both, we support devs that have now access to figma dev mode aswell as those who use figma dev mode with rem

Reply:
I highly vote against this :(

Reply:
ok killed the rem options. people can still use the css variable after all.